test/contrib/rack/middleware_test.rb

```diff
@@ -278,35 +267,8 @@ def test_request_middleware_custom_service
     assert_equal('GET 200', span.resource)
     assert_equal('GET', span.get_tag('http.method'))
     assert_equal('200', span.get_tag('http.status_code'))
-    assert_equal('/', span.get_tag('http.url'))
+    assert_equal('/success/', span.get_tag('http.url'))
     assert_equal(0, span.status)
     assert_nil(span.parent)
   end
 end
-
-class RackBaseTest < Minitest::Test
```

Comment:
Useless tests: we're testing if the Datadog registry works or not, since we don't rely anymore in Rack middleware parameters. Also these tests were execute multiple times (6 to be precise). We moved from 20 tests to 14.